pkg/flag/report_flags.go

```diff
@@ -155,6 +155,10 @@ func (f *ReportFlagGroup) ToOptions(out io.Writer) (ReportOptions, error) {
 	listAllPkgs := getBool(f.ListAllPkgs)
 	output := getString(f.Output)
 
+	if format != "" && !slices.Contains(report.SupportedFormats, format) {
+		return ReportOptions{}, xerrors.Errorf("unknown format: %v", format)
+	}
```

Comment:
Some of the unit tests in `/pkg/flag/report_flags_test.go` use `""` as their format, so I make a wild card for `""`.